javascript-data-model/order-history.js

```diff
@@ -0,0 +1,44 @@
+var orderHistory = [
+  {
+    orderNo: '114-39416898772232',
+    productName: 'JavaScript for impatient programmers',
+    bookAuthor: 'Rauschmayer, Dr.Axel',
+    shipTo: 'JS Masher',
+    price: '$31.55',
+    total: '$34.00',
+    orderPlaced: 'August 4, 2020',
+    orderDelivered: 'August 8, 2020',
+    returnWindow: 'Sep 7, 2020'
+  },
+  {
+    orderNo: '113-9984268-1280257',
+    productName: 'The Timeless Way of Building',
+    bookAuthor: 'Alexander, Christopher',
+    shipTo: 'JS Masher',
+    price: '$41.33',
+    total: '$34.53',
+    orderPlaced: 'July 19, 2020',
+    orderDelivered: 'July 20, 2020',
+    returnWindow: 'Aug 19, 2020'
+  },
+  {
+    orderNo: '114-2875557-9059409',
+    productName: 'Gamecube Controller Adapter',
+    shipTo: 'JS Masher',
+    price: '$15.98',
+    total: '$17.22',
+    orderPlaced: 'July 4, 2020',
+    orderDelivered: 'July 7, 2020',
+    returnWindow: 'Aug 5, 2020'
+  }, {
+    orderNo: '113-2883177-2648248',
+    productName: ['Gamecube Controller', 'The Art of Sql'],
+    bookAuthor: [null, 'Farouly, Stephane'],
```

Comment:
Notice how this entry has arrays for values, but the rest of your entries have strings for values. This is not a consistent data model, which will cause issues if you ever tried to use this data. Make sure that your format matches for all orders.